### DISCUSSION TITLE
riverpod_lint/custom_lint disables Android Studio dart quick assistant

### DISCUSSION THREAD

**Discussion by travisbikkle** [rrousselGit/riverpod#3835] [Q&A]
I followed [riverpod get started](https://riverpod.dev/docs/introduction/getting_started#enabling-riverpod_lintcustom_lint) to add lint to my project, and found that I can't use "wrap with widgets" anymore. 

After remove custom_lint from analysis_options.yaml, problem solved.

before:
<img width="797" alt="image" src="https://github.com/user-attachments/assets/2892409e-9c28-48cc-84ec-3221200fd3d2">

<img width="797" alt="image" src="https://github.com/user-attachments/assets/64fba4a2-ce35-44e3-8a9a-d828dfd64a28">

after:

<img width="491" alt="image" src="https://github.com/user-attachments/assets/47041be4-03df-4b3a-a56a-07dab568d115">

I'm new to flutter, let me know if I did anything wrong, or it is indeed an issue. 

Thanks!

**Comment by YukiAttano** [comment]
Seems to be a problem for custom_lint and nothing riverpod has to do with.
You should file an issue in their repo
